metricbeat/module/vsphere/_meta/config.reference.yml

```
@@ -1,6 +1,6 @@
 - module: vsphere
   enabled: true
-  metricsets: ["datastore", "host", "virtualmachine"]
+  metricsets: ["datastore", "host", "virtualmachine", "datastore_cluster"]
```

```suggestion
  metricsets: ["datastore", "host", "virtualmachine", "datastorecluster"]
```
Shouldn't we keep it same as the metricset's name defined here?
https://github.com/elastic/beats/blob/5f26cc17d620ee7ebeeb0580809fcd6fc2732bd2/metricbeat/module/vsphere/datastore_cluster/datastore_cluster.go#L38-L41